web/app/js/components/LineGraph.jsx

```diff
@@ -133,6 +141,43 @@ export default class LineGraph extends React.Component {
     }
   }
 
+  flashLatestDataPoint() {
+    if (this.props.flashLastDatapoint === false) {
```

Comment:
I think it would be better to use `if (!this.props.flashLastDatapoint)`, since if you don't specify flashLastDatapoint in the props flashLastDatapoint will be undefined, but we also want this to have the same effect as passing "false".

Reply:
yep, much better :)